### PR TITLE
Move to CocoaPods 1.0 

### PR DESCRIPTION
**Pull request by orta**
CP 1.0 is starting to get wrapped up now. Need to ensure that non-trivial apps are working fine.

This greens locally. Will comment inside.

### CODE REVIEW

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -17,7 +17,6 @@
 		0CD4E9B11BC42675005F0913 /* Heart_Shortcut.png in Resources */ = {isa = PBXBuildFile; fileRef = 0CD4E9B01BC42675005F0913 /* Heart_Shortcut.png */; };
 		0CD4E9B51BC43198005F0913 /* AREmbeddedModelPreviewViewControllerTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 0CD4E9B31BC43160005F0913 /* AREmbeddedModelPreviewViewControllerTests.m */; };
 		0CE3C4361B8BB3630092A24F /* EXPMatcher+UINavigationController.m in Sources */ = {isa = PBXBuildFile; fileRef = 0CE3C4341B8BB35C0092A24F /* EXPMatcher+UINavigationController.m */; };
-		2CF5409BF8E46C99AA2FE962 /* Pods_Artsy.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 069CA471836084D186F146A2 /* Pods_Artsy.framework */; };
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: migrating to 1.0 triggers a full reintegration, effectively running `rm -rf Pods` and removing itself from the project entirely, so UUIDs change

File: Artsy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -4641,13 +4641,13 @@
...
-				406A4BAEF8063B0EA2AB20AD /* Check Pods Manifest.lock */,
+				79E285BA175ED99872293B60 /* 📦 Check Pods Manifest.lock */,
 				49BA7DFB1655ABE600C06572 /* Sources */,
 				49BA7DFC1655ABE600C06572 /* Frameworks */,
 				49BA7DFD1655ABE600C06572 /* Resources */,
 				5150685F1BCC176300018352 /* Generate Compilation DB */,
-				3D54B3876CCE10188CF4DCF8 /* Embed Pods Frameworks */,
-				9F607EACDDAF82781666D655 /* Copy Pods Resources */,
+				D094149C23FE012B198AF725 /* 📦 Embed Pods Frameworks */,
+				4F59D9726E617D9849261C9C /* 📦 Copy Pods Resources */,
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: Heh, so cute 👍🏾 @alisoftware

File: Artsy.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -5746,10 +5746,6 @@
 				CLANG_WARN_SUSPICIOUS_IMPLICIT_CONVERSION = NO;
 				CODE_SIGN_ENTITLEMENTS = Artsy/App_Resources/Artsy.entitlements;
 				ENABLE_BITCODE = NO;
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(inherited)",
-					"$(SRCROOT)",
-				);
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: These did nothing, so I removed them

File: Gemfile [hunk 4]
```diff
@@ -1,11 +1,13 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', "~> 0.39"
+# This resolves to either b7 or rc1
+gem 'cocoapods', git: "https://github.com/cocoapods/cocoapods", branch: "mr-fix-resource-bundles"
```
Comment (orta) [inline Gemfile]: We're waiting on https://github.com/CocoaPods/CocoaPods/pull/5049 to merge for this

File: Podfile [hunk 5]
```diff
@@ -91,8 +87,8 @@ target 'Artsy' do
   end
 
   # Facebook
-  pod 'FBSDKCoreKit', '4.9.0-beta2'
-  pod 'FBSDKLoginKit', '4.9.0-beta2'
+  pod 'FBSDKCoreKit', '~> 4.9'
+  pod 'FBSDKLoginKit', '~> 4.9'
```
Comment (orta) [inline Podfile]: Fixes this chain of stuff: https://github.com/CocoaPods/CocoaPods/issues/4723

File: Podfile [hunk 6]
```diff
@@ -82,7 +78,7 @@ target 'Artsy' do
   # Artsy Spec repo stuff
   pod 'Artsy-UIButtons'
   pod 'Artsy+UIColors'
-  pod 'Artsy+UILabels'
+  pod 'Artsy+UILabels', :git => "https://github.com/artsy/Artsy-UILabels.git", :branch => "orta-fonts"
```
Comment (orta) [inline Podfile]: ah this is merged now, will update accordingly

File: Podfile [hunk 7]
```diff
@@ -112,22 +108,25 @@ target 'Artsy' do
...
-  pod 'OCMock'
-  pod 'Forgeries/Mocks', :git => "https://github.com/ashfurrow/Forgeries.git", :branch => "application"
-
-  # Swift pods 🎉
-  pod 'Quick', git: 'https://github.com/Quick/Quick.git'
-  pod 'Nimble', git: 'https://github.com/Quick/Nimble.git'
-  pod 'Nimble-Snapshots', '~> 3.0.0'
+  target 'Artsy Tests' do
+      inherit! :search_paths
+
```
Comment (orta) [inline Podfile]: This has moved _inside_  the `target 'Artsy' do ...` block, as it's inheriting the header search  paths for the Pods from that target. E.g. so the tests can refer to pods that exist in the app.

File: Podfile [hunk 8]
```diff
@@ -149,14 +148,6 @@ post_install do |installer|
     end
   end
 
-
-  app_plist = "Artsy/App_Resources/Artsy-Info.plist"
-  plist_buddy = "/usr/libexec/PlistBuddy"
-  version = `#{plist_buddy} -c "Print CFBundleShortVersionString" #{app_plist}`.strip
-  installer.pods_project.targets.each do |target|
-    `#{plist_buddy} -c "Set CFBundleShortVersionString #{version}" "Pods/Target Support Files/#{target}/Info.plist" > /dev/null 2>&1`
-  end
```
Comment (orta) [inline Podfile]: 1.0 fixes this I believe

File: circle.yml [hunk 9]
```diff
@@ -32,7 +32,7 @@ machine:
 
 dependencies:
   override:
-    - bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4 --retry=3 --without development
+    - bundle check --path=vendor/bundle --without development || bundle install --path=vendor/bundle --jobs=4 --retry=3 --without development
```
Comment (orta) [inline circle.yml]: `bundle check` would never pass, as they have different groups to compare with:

screen shot 2016-04-04 at 09 42 01

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Error</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG changes made</td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by alloy** [comment]
Sweet

**Comment by orta** [comment]
Alright this is green. I've marked the test that keeps crashing the tests as pending, I expect #1370 will deal with it properly, given that I bet there's changes there.

File: Artsy.xcodeproj/project.pbxproj (hunk 2 above)
Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]: :heart_eyes:

**Comment by alloy** [comment]
Looks good to me. Can you rebase?

**Comment by orta** [comment]
Done'd

**Comment by alloy** [comment]
Hello Future!

File: Artsy.xcodeproj/project.pbxproj (hunk 2 above)
Comment (AliSoftware) [inline Artsy.xcodeproj/project.pbxproj]: 😊

**Pull request merged**